test/api_getProductRaw_test.dart

```diff
@@ -83,7 +69,7 @@ void main() {
           result.product!.nutrientLevels!.levels[NutrientLevels.NUTRIENT_SALT],
           Level.MODERATE);
       expect(result.product!.countries,
-          'Belgien,Deutschland,Niederlande,Spanien,Schweiz, en:france');
+          'Bélgica,Francia,Alemania,Países Bajos,España,Suiza');
```

Comment:
Rather peculiar to have now countries in Spanish while they were in German, with a query in German.
Maybe something changed upstream in the backend?
Anyway it would probably be safer to test `countriesTags` or `countriesTagsInLanguages`.

Reply:
In fact we should remove / deprecate the "countries" field from the openfoodfacts-dart package, that field is mostly for internal use, it contains the last value sent by a user, but that value could be in any language. Instead the package must use the countries_tags field.

The same is true for all taxonomized fields like categories or labels (but not yet brands).

Reply:
Feel free to raise an issue for this